packages/devextreme/build/gulp/state_manager/transpile_state_manager_prod.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const through2 = require('through2');
+const path = require('path');
+const fs = require('fs');
+const babel = require('@babel/core');
+const transpileConfig = require('../transpile-config');
+const { STATE_MANAGER_FOLDER_PATH } = require('./constants');
+
+const ERROR_PREFIX = 'Error during transpiling the state manager for prod mode:';
+const PRODUCTION_FILE_NAME = 'production.js';
+
+function transpileStateManagerProd(isEsm) {
+    return through2.obj(function(file, enc, callback) {
+        if (file.path.includes(path.join(STATE_MANAGER_FOLDER_PATH, 'setup_state_manager.js'))) {
```

Comment:
```suggestion
        if (file.path.includes(SETUP_STATE_MANAGER)) {
```
I'd also move this as constant to packages/devextreme/build/gulp/state_manager/constants.js

Reply:
Extracted